### PR TITLE
Add paragraph on current scopes for cursor

### PR DESCRIPTION
**Pull request by bastilian**
This adds a paragraph briefly mentioning and showing the shortcut in to get the scopes for the current cursor position as proposed in #248 under "Finding a Language's Scope Name" in the "Basic customization" section.

### CODE REVIEW

File: content/using-atom/sections/basic-customization.md [hunk 1]
```diff
@@ -225,6 +225,12 @@ The scope name is shown in the settings view for each language. Click on "Packag
 
 ![Finding a language grammar](../../images/python-grammar.png "Finding a grammar's scope name")
 
+Another way to find the scope for a specific language is to open a file of it's kind and press <kbd class="platform-mac">Alt+Cmd+P</kbd><kbd class="platform-windows">Alt+Ctrl+Shift+P</kbd><kbd class="platform-linux">Ctrl+Alt+Shift+P</kbd> to show all scopes for the current position of the cursor.
```
Comment (50Wliu) [inline content/using-atom/sections/basic-customization.md]: `open a file of its kind`
`<kbd class="platform-windows platform-linux">Alt+Ctrl+Shift+P`

**Comment by bastilian** [comment]
Thanks for catching that @50Wliu! Corrected and combined.

**Comment by 50Wliu** [comment]
This looks good to me.

**Comment by lee-dohm** [comment]
Since this is under the "Finding a Language's Scope Name" heading, you should mention that the scope name of the language is always the topmost scope listed. This is because when opening a file they've had open before, their cursor may be on a token that produces three or more scope names, even ones that appear completely unrelated.

**Comment by bastilian** [comment]
It is now mentioning that the top most is the language and the screenshot is updated to illustrate multiple scopes.

**Comment by lee-dohm** [comment]
Thanks a lot @bastilian :grinning:

**Pull request merged**